src/Illuminate/Http/Client/Response.php

```diff
@@ -190,6 +190,16 @@ public function serverError()
         return $this->status() >= 500;
     }
 
+    /**
+     * Determine if the response indicates the request is unauthorized or forbidden.
+     *
+     * @return bool
+     */
+    public function unauthorized()
+    {
+        return $this->status() == 401 || $this->status() == 403;
```

Comment:
This definitely cannot cover 403 which is forbidden and a totally different thing than unauthorized.